docs/technical-report-1.md

```diff
@@ -247,317 +247,317 @@ Limiting nodes to a maximum of one vote would likely be safe if the mean number
+  
+The construction and verification times below are based on benchmarking using the Rust [bls-signatures](https://lib.rs/crates/bls-signatures) package.  
+  
+| Operation                            |       CPU time |  
+| ------------------------------------ | -------------: |  
+| Signing an item                      | 1.369±0.030 ms |  
+| Verifying a single item              | 1.662±0.090 ms |  
+| Verifying an aggregate of 500 items  |    55.3±5.3 ms |  
+| Verifying an aggregate of 1000 items |    100.±20. ms |  
+  
+![Benchmark for verifying an aggregate BLS signature](../images/bls-verification.svg)  
+  
+However, these measurements are not consistent with the experience of Mithril:  
+  
+| Metric                              | Value |  
+| ----------------------------------- | ----- |  
+| Proving time (per vote)             | 70 ms |  
+| Aggregation time (per certificate)  | 1.2 s |  
+| Verification time (per certificate) | 17 ms |  
+  
+  
+#### MUSEN certificate  
+  
+[MUSEN](https://iohk.io/en/research/library/papers/musen-aggregatable-key-evolving-verifiable-random-functions-and-applications/) ("MUlti-Stage key-Evolving verifiable random fuNctions.") builds upon the concept of VRFs by introducing a key-evolving feature and allowing for aggregating VRF outputs. The aggregation capability promises to result in small certificates attesting to the Leios votes, MUSEN suffers from the need to record all of the verification keys (e.g., in a Merkle tree). MUSEN signature are approximately the same size as KES signatures. However, verification times may be too long for Leios. Overall, the benefit of MUSEN's key evolution is minor and its other resource costs may be greater than plain BLS.  
+  
+  
+### Insights regarding voting and certificates  
+  
+1. It is critically important to keep the size of votes small.  
+    1. Large votes can mean large certificates, and votes should also fit in one TCP MTU.  
+    2. Including KES signatures in votes means that votes will be 500-600 B in size.  
+    3. Optimizations might take advantage of the fact that KES periods, currently spanning 1.5 days on the Cardano mainnet, persist for many Leios voting cycles.  
+    4. Using ephemeral keys would make votes smaller, but then a key registration layer would have to be added to the protocol.  
+2. Neither ALBA, BLS, MUSEN, or their ZK variants are clear winners for the best choice of certificate scheme for Leios.  
+    1. Each has at least one strength and one drawback related to certificate size, proof time, or verification time.  
+    2. ALBA is close to being viable if vote size can reduced or if quorum disruption by adversaries with 10% of stake is acceptable.  
```

Comment:
This 10% adversarial stake is within the committee, which may be possible with smaller overall adversarial stake though! We should compute some numbers here to show the probability of having an adversarial quorum of a given size w.r.t. the total adversarial stake.

Reply:
Yes, the 10% isn't exact. The fluctuations around that are small for a committee of 500, as discussed in the "Committee size and quorum requirement" section.

Fixed in 0aa54aace59c5927d485901e1d6070c5b5f42864.